### ISSUE TITLE
Does this app run in the background?

### ISSUE THREAD

**Issue by illuzionz14** [PebbleSeizureDetect/PebbleSeizureDetect#14]
Thanks for putting this app together Ryan.  Just wondering if this app runs in the background?  Seems like the app would have a chance to exit if the buttons are accidentally pressed.  Any info would be much appreciated .. thanks

**Comment by bygRyan** [comment]
Unfortunately no, it doesn't run in the background :(

**Comment by jones139** [comment]
I don't think it is possible to have it run in the background, because background apps can not use the bluetooth radio to communicate with the phone.
The way I got round the probem of button pressing for OpenSeizureDetector is to have the phone app check that it receives data from the watch  when it expects it.  If it does not receive data on time, it re-starts the watch app.
